### ISSUE TITLE
Wipe tower out of print area

### ISSUE THREAD

**Issue by pastaclub** [bobotmedia/3DWOX1#2]
Hi, are these profiles for the Sindoh slicer or a different one? And for 3DWOX 1, right?

I tried to import the PETG profile as "custom profile" and the slicer complains that the wipe tower is outside of the printing area. However, the "wipe tower" tab is disabled in the settings.

And about the 0.5mm extrusion width: I saw in the ini file that you have set the nozzle diameter to 0.5mm. Is this intended to work with the default 0.4mm nozzle (just more extrusion) or would I actually need a 0.5mm nozzle for this profile?

**Comment by bobotmedia** [comment]
Yes, all the profiles listed as 3DWOX1 are for that machine only.  Scroll down to the README and it says if the profile is for Sindoh Desktop slicer or Simplify3D slicer.  In general, if the file extension is .fff then it is for Simplify3D, and if .ini it is for Sindoh slicer.  Not sure about the wipe tower....maybe make sure you are running the newest version of Sindoh Desktop slicing software.   Yes, I run the 0.5mm profile using the standard 0.4mm nozzle that came with the machine.
